src/main/resources/wa-task-initiation-sscs-benefit.dmn

```diff
@@ -433,6 +478,41 @@ else 0</text>
           <text>"reviewUrgentHearingRequest"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_114m1jf">
+        <inputEntry id="UnaryTests_01j9v2i">
+          <text>"tcwReferToJudge"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0phwa4k">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0vwit8f">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0mfdwpj">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0zhspby">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_11bt77z">
+          <text>"preHearingWork"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0wid6ky">
+          <text>"referredByTcwPreHearing"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_01iy7nj">
+          <text>"Referred By TCW Pre Hearing"</text>
```

Comment:
Task name just needs to be "Referred By TCW"

Reply:
Thanks, corrected.